src/api_client.rs

```diff
@@ -203,20 +203,39 @@ impl Client {
     }
```

Comment:
can even refactor `get_all_committees` to call `self.get_committees(id, CommitteeFilter::default())`

... or even just delete `get_all_committees` for now -- this may have been a little over-eager when I wrote it out

Reply:
ok, refactored `get_all_committees()`- can delete later if you decide that's best

Reply:
thank you :)